sound/soc/sof/intel/Kconfig

```diff
@@ -61,17 +61,12 @@ if SND_SOC_SOF_INTEL_ACPI
 
 config SND_SOC_SOF_BAYTRAIL_SUPPORT
 	bool "SOF support for Baytrail, Braswell and Cherrytrail"
-	depends on SND_SST_ATOM_HIFI2_PLATFORM_ACPI=n
 	help
 	  This adds support for Sound Open Firmware for Intel(R) platforms
 	  using the Baytrail, Braswell or Cherrytrail processors.
-	  This option is mutually exclusive with the Atom/SST and Baytrail
-	  legacy drivers. If you want to enable SOF on Baytrail/Cherrytrail,
-	  you need to deselect those options first.
-	  SOF does not support Baytrail-CR for now, so this option is not
-	  recommended for distros. At some point all legacy drivers will be
-	  deprecated but not before all userspace firmware/topology/UCM files
-	  are made available to downstream distros.
+	  This option can coexit with the Atom legacy drivers, which will be
+	  deprecated at some point. Existing userspace firmware/topology/UCM
```

Comment:
is this how we refer to them - "userspace firmware" and "userspace topology?"

Reply:
the previous wording has been there for a while. I am open to suggestions to improve, but all these files are part of the 'we don't break userspace' set of data stored on the filesystem that need to work with a new kernel.